### ISSUE TITLE
softmax act on output, but not a cross entropy error func, warning needed

### ISSUE THREAD

**Issue by dsblank** [Calysto/conx#93]

**Comment by dsblank** [comment]
Fixed in b3413b1
